### PR TITLE
[DOCFIX] [branch-1.6] Add getting started doc on a distributed cluster

### PR DESCRIPTION
**Pull request by ifcharming**

### CODE REVIEW

File: docs/en/Getting-Started-on-Cluster.md [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+
+First, [download the Alluxio release](http://www.alluxio.org/download) locally. You can
+download the latest {{site.ALLUXIO_RELEASED_VERSION}} release pre-built for various versions of
+Hadoop from the [Alluxio download page](http://www.alluxio.org/download).
+
+Second, `scp` the release tarball to all the Linux nodes. Note that, feel free to use scriptable way (such as Ansible)
+to distribute the tarball and automate the following process on multiple machines. In this example,
+manually commands are shown and most of them should be executed on all the nodes.
+
+Next, you can unpack the download with the following commands on all the nodes. Your filename may be different
```
Comment (yupeng9) [inline docs/en/Getting-Started-on-Cluster.md]: file name

File: docs/en/Getting-Started-on-Cluster.md [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+## Configuring Alluxio
+
+Before we start Alluxio, we have to configure it. Please ensure all the subsequent commands are run on all the Linux nodes.
+
+In the `${ALLUXIO_HOME}/conf` directory, create the `conf/alluxio-site.properties` configuration
+file from the template.
+
+```bash
+$ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
+```
```
Comment (yupeng9) [inline docs/en/Getting-Started-on-Cluster.md]: User also needs to modify the `conf/workers` file?

File: docs/en/Getting-Started-on-Cluster.md [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+
+The [Quick Start Guide](Getting-Started.html) shows how to install Alluxio locally on a single machine.
+In this advanced quick start guide, we will install Alluxio on a 3-node Linux cluster, configure a
+AWS S3 bucket as UFS (Under File System), and perform basic tasks with the data in Alluxio.
+
+During this guide, you will:
+
+* Download and configure Alluxio in a Linux cluster
+* Configuration for AWS
+* Validating Alluxio environment
```
[the remark this answers is not in the document]
Comment (ifcharming) [reply docs/en/Getting-Started-on-Cluster.md]: done

File: docs/en/Getting-Started-on-Cluster.md [hunk 4]
```diff
@@ -0,0 +1,334 @@
...
+* Set up a distributed storage as UFS
+* Start Alluxio on multiple nodes
+* Perform basic tasks via Alluxio Shell
+* **[Bonus]** Run sample Hadoop MapReduce job in Alluxio cluster
+* **[Bonus]** Run sample Apache Spark job in Alluxio cluster
+* Shutdown Alluxio
+
+**[Required]** In this documentation, we will use AWS S3 as the example distributed storage as Alluxio under file system.
+Please make sure you have an [AWS account with an access key id and secret access key](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html),
+you will be able to use S3 in this guide. If you don't have a S3 account, you can also use an existing HDFS cluster as the under
```
[the remark this answers is not in the document]
Comment (ifcharming) [reply docs/en/Getting-Started-on-Cluster.md]: done

File: docs/en/Getting-Started-on-Cluster.md [hunk 5]
```diff
@@ -0,0 +1,334 @@
...
+file system. The "Set up a distributed storage as UFS" section will be slightly different.
```
[the remark this answers is not in the document]
Comment (ifcharming) [reply docs/en/Getting-Started-on-Cluster.md]: done, thx

File: docs/en/Getting-Started-on-Cluster.md [hunk 6]
```diff
@@ -0,0 +1,334 @@
...
+
+**Note**  This guide is meant for you to quickly start interacting with an Alluxio system, in a distributed environment.
```
[the remark this answers is not in the document]
Comment (ifcharming) [reply docs/en/Getting-Started-on-Cluster.md]: done

File: docs/en/Getting-Started-on-Cluster.md (hunk 1 above)
Comment (ifcharming) [reply docs/en/Getting-Started-on-Cluster.md]: done

File: docs/en/Getting-Started-on-Cluster.md (hunk 2 above)
Comment (ifcharming) [reply docs/en/Getting-Started-on-Cluster.md]: I don't think so. Registered worker will be added to `conf/workers` automatically.

File: docs/en/Getting-Started-on-Cluster.md [hunk 7]
```diff
@@ -0,0 +1,334 @@
...
+{:toc}
+
+The [Quick Start Guide](Getting-Started.html) shows how to install Alluxio locally on a single machine.
+In this advanced quick start guide, we will install Alluxio on a 3-node Linux cluster, configure a
+AWS S3 bucket as UFS (Under File System), and perform basic tasks with the data in Alluxio.
+
+During this guide, you will:
+
+* Download and configure Alluxio in a Linux cluster
+* Configuration for AWS
```
Comment (gpang) [inline docs/en/Getting-Started-on-Cluster.md]: This should be a verb, to be consistent with other bullet points.

File: docs/en/Getting-Started-on-Cluster.md [hunk 8]
```diff
@@ -0,0 +1,334 @@
...
+* Validate Alluxio environment
+* Set up a distributed storage as UFS
+* Start Alluxio on multiple nodes
+* Perform basic tasks via Alluxio Shell
+* **[Bonus]** Run sample Hadoop MapReduce job in Alluxio cluster
+* **[Bonus]** Run sample Apache Spark job in Alluxio cluster
+* Shutdown Alluxio
+
+**[Required]** In this documentation, we will use AWS S3 as the example distributed storage as Alluxio under file system.
+Please make sure you have an [AWS account with an access key id and secret access key](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html),
```
Comment (gpang) [inline docs/en/Getting-Started-on-Cluster.md]: The link is for SQS. Is there a more general AWS link to point to?

File: docs/en/Getting-Started-on-Cluster.md [hunk 9]
```diff
@@ -0,0 +1,334 @@
...
+
+For the following quick start guide, you will need:
+
+* 3-nodes Linux cluster
+* [Java 7 or newer](Java-Setup.html)
+* AWS account and keys
+
+### Setup SSH (Linux)
+
+Please make sure on all the nodes, ssh is enabled.
```
Comment (gpang) [inline docs/en/Getting-Started-on-Cluster.md]: Does passwordless ssh also need to be set up?

File: docs/en/Getting-Started-on-Cluster.md [hunk 10]
```diff
@@ -0,0 +1,334 @@
...
+
+## Downloading Alluxio
+
+First, [download the Alluxio release](http://www.alluxio.org/download) locally. You can
+download the latest {{site.ALLUXIO_RELEASED_VERSION}} release pre-built for various versions of
+Hadoop from the [Alluxio download page](http://www.alluxio.org/download).
+
+Second, `scp` the release tarball to all the Linux nodes. Note that, feel free to use scriptable way (such as Ansible)
```
Comment (gpang) [inline docs/en/Getting-Started-on-Cluster.md]: Should './bin/alluxio copyDir` be used instead of `scp`?

File: docs/en/Getting-Started-on-Cluster.md (hunk 2 above)
Comment (gpang) [reply docs/en/Getting-Started-on-Cluster.md]: `conf/workers` is not modified by Alluxio. The admin has to add the hostnames to that file, so the Alluxio scripts know which hosts to start workers on. The same goes for masters.

File: docs/en/Getting-Started-on-Cluster.md [hunk 11]
```diff
@@ -0,0 +1,334 @@
...
+$ tar -xzf alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-bin.tar.gz
+$ cd alluxio-{{site.ALLUXIO_RELEASED_VERSION}}
+```
+
+This will create a directory `alluxio-{{site.ALLUXIO_RELEASED_VERSION}}` with all of the Alluxio
+source files and Java binaries.
+
+## Configuring Alluxio
+
+Before we start Alluxio, we have to configure it. Please ensure all the subsequent commands are run on all the Linux nodes.
```
Comment (gpang) [inline docs/en/Getting-Started-on-Cluster.md]: Can we configure `conf/workers` and use `copyDir` instead?

File: docs/en/Getting-Started-on-Cluster.md [hunk 12]
```diff
@@ -0,0 +1,334 @@
...
+
+Will only run validation tasks that check your system resource limits, on all the nodes.
+
+You can check out [this page](Developer-Tips.html) for detailed usage information regarding this command.
+
+## Setting up a distributed storage as UFS
+
+The [local Getting-Started guide](Getting-Started.html) shows the steps of how to start Alluxio with default local file system.
+In a distributed cluster, Alluxio requires a distributed storage system as Under File System.
+In this doc, we use AWS S3 as an example here. If you want to use an existing distributed HDFS as UFS, the setup is similar.
```
Comment (gpang) [inline docs/en/Getting-Started-on-Cluster.md]: I would either remove the HDFS part, or actually point to an HDFS setup page.

File: docs/en/Getting-Started-on-Cluster.md [hunk 13]
```diff
@@ -0,0 +1,334 @@
...
+
+Also, in preparation for using S3 with Alluxio, create a bucket (or use an existing bucket). You
+should also note the directory you want to use in that bucket, either by creating a new directory in
+the bucket, or using an existing one. For the purposes of this guide, the S3 bucket name is called
+`S3_BUCKET`, and the directory in that bucket is called `S3_DIRECTORY`.
+
+You need to configure Alluxio to use S3 as its under storage system by modifying
+`conf/alluxio-site.properties`. The first modification is to specify an **existing** S3
+bucket and directory as the under storage system. You specify it by modifying
+`conf/alluxio-site.properties` to include:
```
Comment (gpang) [inline docs/en/Getting-Started-on-Cluster.md]: Where? on the master?

File: docs/en/Getting-Started-on-Cluster.md [hunk 14]
```diff
@@ -0,0 +1,334 @@
...
+
+If you want to enable more advanced feature for S3 UFS, please refer to this [documentation](Configuring-Alluxio-with-S3.html).
+
+## Starting Alluxio
+
+Next, we will format Alluxio in preparation for starting Alluxio. The following command will format
+the Alluxio journal and the worker storage directory in preparation for the master and worker to
+start.
+
+Please make sure this command is run on all the nodes.
```
Comment (gpang) [inline docs/en/Getting-Started-on-Cluster.md]: If `conf/workers`, and ssh are configured, then you only have to run this on the master.

File: docs/en/Getting-Started-on-Cluster.md [hunk 15]
```diff
@@ -0,0 +1,334 @@
...
+
+```bash
+$ ./bin/alluxio format
+```
+
+Now, we can start Alluxio! In this doc, we will start one Alluxio Master and two Alluxio Workers.
+
+On one of the node (naming it Master Node), run the following command to start Alluxio Master:
```
Comment (gpang) [inline docs/en/Getting-Started-on-Cluster.md]: If `conf/workers` is setup, then you can just run `./bin/alluxio-start.sh all`

File: docs/en/Getting-Started-on-Cluster.md [hunk 16]
```diff
@@ -0,0 +1,334 @@
...
+$ ./bin/alluxio fs copyFromLocal LICENSE /LICENSE
+Copied LICENSE to /LICENSE
+```
+
+After copying the `LICENSE` file, we should be able to see it in Alluxio. List the files in
+Alluxio with the command:
+
+```bash
+$ ./bin/alluxio fs ls /
+26.22KB   06-20-2016 11:30:04:415  In Memory      /LICENSE
```
Comment (gpang) [inline docs/en/Getting-Started-on-Cluster.md]: This format looks old?

File: docs/en/Getting-Started-on-Cluster.md (hunk 7 above)
Comment (ifcharming) [reply docs/en/Getting-Started-on-Cluster.md]: done

File: docs/en/Getting-Started-on-Cluster.md (hunk 8 above)
Comment (ifcharming) [reply docs/en/Getting-Started-on-Cluster.md]: done, thx

File: docs/en/Getting-Started-on-Cluster.md (hunk 9 above)
Comment (ifcharming) [reply docs/en/Getting-Started-on-Cluster.md]: I add more docs regarding setting up password-less ssh and use `copyDir` as alternatives.

File: docs/en/Getting-Started-on-Cluster.md (hunk 16 above)
Comment (ifcharming) [reply docs/en/Getting-Started-on-Cluster.md]: updated

File: docs/en/Getting-Started-on-Cluster.md [hunk 17]
```diff
@@ -124,7 +124,7 @@ Will only run validation tasks that check your system resource limits, on all th
 
 You can check out [this page](Developer-Tips.html) for detailed usage information regarding this command.
 
-## Set up a distributed storage as UFS
+## Setting up a distributed storage as UFS
```
Comment (yupeng9) [inline docs/en/Getting-Started-on-Cluster.md]: Use `Set up` for consistency

File: docs/en/Getting-Started-on-Cluster.md (hunk 2 above)
Comment (yupeng9) [reply docs/en/Getting-Started-on-Cluster.md]: Right. That's my understanding too

File: docs/en/Getting-Started-on-Cluster.md (hunk 12 above)
Comment (ifcharming) [reply docs/en/Getting-Started-on-Cluster.md]: done

File: docs/en/Getting-Started-on-Cluster.md (hunk 13 above)
Comment (ifcharming) [reply docs/en/Getting-Started-on-Cluster.md]: all the nodes, updated

File: docs/en/Getting-Started-on-Cluster.md (hunk 17 above)
Comment (ifcharming) [reply docs/en/Getting-Started-on-Cluster.md]: done

**Comment by ifcharming** [comment]
@yupeng9  @gpang thanks for the comments. I'm open to switch to `conf/workers` and `CopyDir`.

I was describing the manual process here, to demo how to start each module individually and how they work together. In some user's environment, opening password-less ssh and SudoMount might be hard. In that case, user might want to customize their way to configure Alluxio manually. I've add the section that lists the alternative ways to automate those steps (and links to other sub-docs).

File: docs/en/Getting-Started-on-Cluster.md (hunk 2 above)
Comment (yupeng9) [reply docs/en/Getting-Started-on-Cluster.md]: ping, I don't think this demo can work without the `conf/workers` set up. Could you double check it?

Comment (ifcharming) [reply docs/en/Getting-Started-on-Cluster.md]: It should still work with starting everything independently. But to make it explicit, I've added a `TIPS` section to mention the more scalable way of using `conf/workers`, `copyDir` and `start all`

File: docs/en/Getting-Started-on-Cluster.md [hunk 18]
```diff
@@ -0,0 +1,348 @@
...
+## Download Alluxio
+
+First, [download the Alluxio release](http://www.alluxio.org/download) locally. You can
+download the latest {{site.ALLUXIO_RELEASED_VERSION}} release pre-built for various versions of
+Hadoop from the [Alluxio download page](http://www.alluxio.org/download).
+
+Second, `scp` the release tarball to all the Linux nodes. Note that, feel free to use scriptable way (such as Ansible)
+to distribute the tarball and automate the following process on multiple machines. In this documentation,
+manually commands are shown and most of them should be executed on all the nodes, to help user better understand
+how Alluxio master and Alluxio workers start individually and how they work together.
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: maybe break this sentence up, I found it hard to follow

File: docs/en/Getting-Started-on-Cluster.md [hunk 19]
```diff
@@ -0,0 +1,348 @@
...
+**AWS_SECRET_ACCESS_KEY** with your AWS secret access key. Now, Alluxio is fully configured for the
+rest of this guide.
+
+## Validate Alluxio environment
+
+Before starting Alluxio, you might want to make sure that your system environment is ready for running
+Alluxio services. You can run the following command to validate environment on all nodes:
+
+```bash
+$ ./bin/alluxio validateEnv all
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: This requires `conf/workers` to be set up, but the guide doesn't assume that

File: docs/en/Getting-Started-on-Cluster.md [hunk 20]
```diff
@@ -0,0 +1,348 @@
...
+```
+
+You can also make the command run only specific validation task. For example,
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: task -> tasks

File: docs/en/Getting-Started-on-Cluster.md [hunk 21]
```diff
@@ -0,0 +1,348 @@
...
+```
+
+Will only run validation tasks that check your system resource limits, on all the nodes.
+
+You can check out [this page](Developer-Tips.html) for detailed usage information regarding this command.
+
+## Set up a distributed storage as UFS
+
+The [local Getting-Started guide](Getting-Started.html) shows the steps of how to start Alluxio with default local file system.
+In a distributed cluster, Alluxio requires a distributed storage system as Under File System.
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: we don't usually capitalize "under file system"

File: docs/en/Getting-Started-on-Cluster.md [hunk 22]
```diff
@@ -0,0 +1,348 @@
...
+In this doc, we use AWS S3 as an example here. If you want to use an existing distributed HDFS as UFS,
+please refer to this [doc](Configuring-Alluxio-with-HDFS.html).
+
+Also, in preparation for using S3 with Alluxio, create a bucket (or use an existing bucket). You
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: Remove "Also,"

File: docs/en/Getting-Started-on-Cluster.md [hunk 23]
```diff
@@ -0,0 +1,348 @@
...
+should also note the directory you want to use in that bucket, either by creating a new directory in
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: "Choose a directory in the S3 bucket to use as the under storage, creating it if it doesn't already exist"

File: docs/en/Getting-Started-on-Cluster.md [hunk 24]
```diff
@@ -0,0 +1,348 @@
...
+`conf/alluxio-site.properties` to include:
+
+```
+alluxio.underfs.address=s3a://S3_BUCKET/S3_DIRECTORY
+```
+
+Note that in the previous `Configuration for AWS` section, the required AWS credential is already setup on
+all the Alluxio nodes. This ensures those nodes have the access to this S3 bucket.
+
+If you want to enable more advanced feature for S3 UFS, please refer to this [documentation](Configuring-Alluxio-with-S3.html).
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: feature -> features

File: docs/en/Getting-Started-on-Cluster.md [hunk 25]
```diff
@@ -0,0 +1,348 @@
...
+
+## Start Alluxio
+
+Next, we will format Alluxio in preparation for starting Alluxio. The following command will format
+the Alluxio journal and the worker storage directory in preparation for the master and worker to
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: we don't need to say "in preparation for starting Alluxio" in two consecutive sentences

File: docs/en/Getting-Started-on-Cluster.md [hunk 26]
```diff
@@ -0,0 +1,348 @@
...
+start.
+
+Please make sure this command is run on all the nodes.
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: This only needs to be run on the master node

File: docs/en/Getting-Started-on-Cluster.md [hunk 27]
```diff
@@ -0,0 +1,348 @@
...
+
+```bash
+$ ./bin/alluxio format
+```
+
+Now, we can start Alluxio! In this doc, we will start one Alluxio Master and two Alluxio Workers.
+
+On one of the node (naming it Master Node), run the following command to start Alluxio Master:
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: node -> nodes

File: docs/en/Getting-Started-on-Cluster.md [hunk 28]
```diff
@@ -0,0 +1,348 @@
...
+
+```bash
+$ ./bin/alluxio-start.sh master
+```
+
+Congratulations! Alluxio master is now up and running! You can visit
+[http://ALLUXIO_MASTER_HOSTNAME:19999](http://ALLUXIO_MASTER_HOSTNAME:19999) to see the status of the Alluxio master.
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: they probably won't actually use "ALLUXIO_MASTER_HOSTNAME" for their master's hostname, so I suggest not making this a hyperlink. Same for ALLUXIO_WORKER1_HOSTNAME below

File: docs/en/Getting-Started-on-Cluster.md [hunk 29]
```diff
@@ -0,0 +1,348 @@
...
+
+In a few seconds, Alluxio workers will register with the Alluxio master.
+You can visit [http://ALLUXIO_WORKER1_HOSTNAME:30000](http://ALLUXIO_WORKER1_HOSTNAME:30000)
+to see the status of the Alluxio worker.
+
+## Use the Alluxio Shell
+
+Now that Alluxio is running, we can examine the Alluxio file system with the
+[Alluxio shell](Command-Line-Interface.html). The Alluxio shell enables many command-line operations
+for interacting with Alluxio. You can ssh to the Alluxio Master node, and invoke the Alluxio shell with
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: for interacting -> to interact

Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: no comma after "Alluxio Master node"

File: docs/en/Getting-Started-on-Cluster.md [hunk 30]
```diff
@@ -0,0 +1,348 @@
...
+After copying the `LICENSE` file, we should be able to see it in Alluxio. List the files in
+Alluxio with the command:
+
+```bash
+$ ./bin/alluxio fs ls /
+drwxr-xr-x   ubuntu    ubuntu    26.22KB   09-22-2017 09:30:08:781  In Memory      /LICENSE
+```
+
+The output shows the file that exists in Alluxio, as well as some other useful information, like the
+size of the file, the date it was created, and the in-Alluxio status of the file.
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: "... and how much of the file is cached in Alluxio"

File: docs/en/Getting-Started-on-Cluster.md [hunk 31]
```diff
@@ -0,0 +1,348 @@
...
+
+You can also view the contents of the file through the Alluxio shell. The `cat` command will print
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: remove "also"

File: docs/en/Getting-Started-on-Cluster.md [hunk 32]
```diff
@@ -0,0 +1,348 @@
...
+$ ./bin/alluxio fs cat /LICENSE
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+...
+```
+
+With this UFS configuration, Alluxio uses the specified S3 bucket as its UnderFileSystem (UFS).
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: UnderFileSystem -> under file system

File: docs/en/Getting-Started-on-Cluster.md [hunk 33]
```diff
@@ -0,0 +1,348 @@
...
+We can check whether this file exists in the S3 bucket.
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: Combine with next paragraph

File: docs/en/Getting-Started-on-Cluster.md [hunk 34]
```diff
@@ -0,0 +1,348 @@
...
+
+Please make sure all the environment and configuration are set up on all the nodes.
+
+## Shutdown Alluxio
+
+Once you are done with interacting with your Alluxio cluster installation, you can stop Alluxio with
+the following command:
+
+```bash
+$ ./bin/alluxio-stop.sh all
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: this assumes `conf/workers`

**Comment by ifcharming** [comment]
@aaudiber thank you for the review. I've address all your comments, can you PTAL?

File: docs/en/Getting-Started-on-Cluster.md (hunk 33 above)
Comment (aaudiber) [reply docs/en/Getting-Started-on-Cluster.md]: I meant to combine "We can check whether this file exists in the S3 bucket." - the sentence is awkward as its own paragraph because it isn't true, and requires the next paragraph to make sense.

File: docs/en/Getting-Started-on-Cluster.md [hunk 35]
```diff
@@ -155,30 +155,29 @@ alluxio.underfs.address=s3a://S3_BUCKET/S3_DIRECTORY
...
 
-If you want to enable more advanced feature for S3 UFS, please refer to this [documentation](Configuring-Alluxio-with-S3.html).
+If you want to enable more advanced features for S3 UFS, please refer to this [documentation](Configuring-Alluxio-with-S3.html).
 
 ## Start Alluxio
 
 Next, we will format Alluxio in preparation for starting Alluxio. The following command will format
-the Alluxio journal and the worker storage directory in preparation for the master and worker to
-start.
+the Alluxio journal and the worker storage directory.
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: we don't use it to format the worker storage directory any more in this doc

File: docs/en/Getting-Started-on-Cluster.md [hunk 36]
```diff
@@ -50,19 +50,19 @@ Please make sure on all the nodes, ssh is enabled.
...
 will make distributed copy and configuration easier. In this doc, we use the example of manual
 downloading and starting Alluxio on multiple nodes, for demo purpose only. The manual process here
-can be much simplified and automated using `CopyDir` or `Ansible` scripts. Please refer to this [doc](Running-Alluxio-on-a-Cluster.html]
+can be much simplified and automated using `CopyDir` or Ansible scripts. Please refer to this [doc](Running-Alluxio-on-a-Cluster.html]
 and [Running-Alluxio-on-EC2](Running-Alluxio-on-EC2.html] for the alternatives.
 
 ## Download Alluxio
 
-First, [download the Alluxio release](http://www.alluxio.org/download) locally. You can
+First, [download the Alluxio release](https://www.alluxio.org/download) locally. You can
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: the alluxio.org site doesn't currently support https, only use https for links which support it (should be everything non-alluxio that we link to)

File: docs/en/Getting-Started-on-Cluster.md (hunk 19 above)
Comment (aaudiber) [reply docs/en/Getting-Started-on-Cluster.md]: ping

File: docs/en/Getting-Started-on-Cluster.md (hunk 35 above)
Comment (ifcharming) [reply docs/en/Getting-Started-on-Cluster.md]: good catch, done, thanks

File: docs/en/Getting-Started-on-Cluster.md (hunk 36 above)
Comment (ifcharming) [reply docs/en/Getting-Started-on-Cluster.md]: done

**Comment by ifcharming** [comment]
@aaudiber thanks! PTAL?

File: docs/en/Getting-Started-on-Cluster.md (hunk 19 above)
Comment (ifcharming) [reply docs/en/Getting-Started-on-Cluster.md]: done

File: docs/en/Getting-Started-on-Cluster.md (hunk 33 above)
Comment (ifcharming) [reply docs/en/Getting-Started-on-Cluster.md]: done

**Comment by ifcharming** [comment]
Jenkins, test this please

**Comment by ifcharming** [comment]
Build failure is not related with this DOCFIX PR. merging
